docs/release/3.4/release-3-4-11.md

```diff
@@ -0,0 +1,103 @@
+OSG Software Release 3.4.11
+===========================
+
+**Release Date**: 2018-05-01
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   [Singularity 2.5.0](https://github.com/singularityware/singularity/releases/tag/2.5.0)
+    - This release includes fixes for several high and medium severity security issues.
+      It also contains a whole slew of bug fixes including the much awaited docker aufs whiteout file fix.
```

Comment:
Actually, "whole slew of" is probably too informal, and maybe also overstates it.  How about just "a number of".

Reply:
I just cut and pasted that out of their announcement.

Reply:
Alright if you must, but we don't have to be as informal as they are.

Reply:
Not a problem, I update it to make us look more classy.